frontend/src/app/modules/auth/sign-up/sign-up.component.ts

```diff
@@ -103,7 +103,8 @@ export class SignUpComponent extends BaseComponent implements OnInit {
         this.authService
             .signUp(this.email.value, this.password.value)
             .pipe(switchMap(() => this.createUser().pipe(this.untilThis)))
-            .subscribe(() => {
+            .subscribe((user) => {
+                localStorage.setItem('user', JSON.stringify(user));
```

Comment:
Are you sure model is the same?
And please, don't work with localStorage in components, only in services

Reply:
fixed